opentracing/tracer.py

```diff
@@ -53,34 +53,33 @@ def scope_manager(self):
 
     def start_active_span(self,
                           operation_name,
-                          finish_on_close,
                           child_of=None,
                           references=None,
                           tags=None,
                           start_time=None,
-                          ignore_active_span=False):
+                          ignore_active_span=False,
+                          finish_on_close=True):
         """Returns a newly started and activated `Scope`.
 
         The returned `Scope` supports with-statement contexts. For example:
 
-            with tracer.start_active_span('...', False) as scope:
+            with tracer.start_active_span('...') as scope:
                 scope.span.set_tag('http.method', 'GET')
                 do_some_work()
-            # Span is not finished outside the `Scope` `with`.
+            # Span is finished outside the `Scope` `with`.
```

Comment:
this is an odd sentence.

Reply:
What about changing it to: 

Span.finish() is called as part of Scope deactivation through the `with` statement.

?

Reply:
+1